### PR TITLE
[FLINK-35773][docs]  Document s5cmd

### PR DESCRIPTION
**Pull request by pnowojski**
## What is the purpose of the change

Document s5cmd. Depends on #25228, please take a look here just at the last commit `[FLINK-35773] Document s5cmd`

## Brief change log

N/A

## Verifying this change

N/A

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / **docs** / JavaDocs / not documented)

### CODE REVIEW

**Review by StefanRRichter — APPROVED** [review]
lgtm

**Review by rkhachatryan — APPROVED** [review]
Thanks for the PR,
LGTM apart from a couple of minor remarks.

And I guess we also need to update `zh` version?

File: docs/content/docs/deployment/filesystems/s3.md [hunk 1]
```diff
@@ -164,4 +164,38 @@ The `s3.entropy.key` defines the string in paths that is replaced by the random
...
+[Benchmark results](https://cwiki.apache.org/confluence/display/FLINK/FLIP-444%3A+Native+file+copy+support) are showing that `s5cmd` can be over 2 times more CPU efficient. 
+Which means either using half the CPU to upload or download the same set of files, or doing that twice as fast with the same amount of available CPU.
+
+In order to use this feature, the `s5cmd` binary has to be present and accessible to the Flink's task managers, for example via embedding it in the used docker image.
+Secondly the path to the `s5cmd` has to be configured via:
+```yaml
+s3.s5cmd.path: /path/to/the/s5cmd
+```
+
+The remaining configuration options (with their default value listed below) are:
```
[the remark this answers is not in the document]
Comment (pnowojski) [reply docs/content/docs/deployment/filesystems/s3.md:179]: I'm doing that in a section below. Or do you mean something else?

[the remark this answers is not in the document]
Comment (rkhachatryan) [reply docs/content/docs/deployment/filesystems/s3.md:179]: Yes, that's what I meant :+1:

**Review by rkhachatryan — APPROVED** [review]

**Pull request merged**